util/network-devp2p/src/lib.rs

```diff
@@ -60,45 +60,49 @@
 //TODO: use Poll from mio
 #![allow(deprecated)]
 
+extern crate ansi_term;
+#[cfg(test)] #[macro_use]
+extern crate assert_matches;
+extern crate bytes;
+extern crate crypto as rcrypto;
+#[cfg(test)]
+extern crate env_logger;
+#[macro_use]
+extern crate error_chain;
 extern crate ethcore_io as io;
-extern crate parity_bytes;
-extern crate parity_crypto as crypto;
+extern crate ethcore_network as network;
 extern crate ethereum_types;
-extern crate parking_lot;
-extern crate mio;
-extern crate tiny_keccak;
-extern crate crypto as rcrypto;
-extern crate rand;
-extern crate ansi_term; //TODO: remove this
-extern crate rustc_hex;
-extern crate igd;
-extern crate libc;
-extern crate slab;
 extern crate ethkey;
-extern crate rlp;
-extern crate bytes;
-extern crate parity_path;
-extern crate ethcore_network as network;
+extern crate igd;
 extern crate ipnetwork;
 extern crate keccak_hash as hash;
-extern crate serde;
-extern crate serde_json;
-extern crate parity_snappy as snappy;
-extern crate lru_cache;
-
-#[macro_use]
-extern crate error_chain;
+extern crate libc;
 #[macro_use]
 extern crate log;
+extern crate lru_cache;
+extern crate mio;
+extern crate parity_bytes;
+extern crate parity_crypto as crypto;
+extern crate parity_path;
+extern crate parity_snappy as snappy;
+extern crate parking_lot;
+extern crate rand;
+extern crate rlp;
+//TODO: remove this
```

Comment:
was it referring to `ansi_term`?

Reply:
Yeah (although it sort of applies to `rustc-hex` version 1 as well!)